l2-contracts/contracts/bridge/L2NativeTokenVault.sol

```diff
@@ -95,53 +97,60 @@ contract L2NativeTokenVault is IL2NativeTokenVault, Ownable2StepUpgradeable {
         emit L2TokenBeaconUpdated(_l2TokenBeacon, _l2TokenProxyBytecodeHash);
     }
 
-    /// @notice Mints the wrapped asset during shared bridge deposit finalization.
+    /// @notice Used when the chain receives a transfer from L1 Shared Bridge and correspondingly mints the asset.
     /// @param _chainId The chainId that the message is from.
     /// @param _assetId The assetId of the asset being bridged.
-    /// @param _transferData The abi.encoded transfer data.
-    function bridgeMint(uint256 _chainId, bytes32 _assetId, bytes calldata _transferData) external payable override {
+    /// @param _data The abi.encoded transfer data.
+    function bridgeMint(uint256 _chainId, bytes32 _assetId, bytes calldata _data) external payable override onlyBridge {
         address token = tokenAddress[_assetId];
-        (address _l1Sender, uint256 _amount, address _l2Receiver, bytes memory erc20Data, address originToken) = abi
-            .decode(_transferData, (address, uint256, address, bytes, address));
-        address expectedToken = l2TokenAddress(originToken);
+        (
+            uint256 _amount,
+            address _l1Sender,
```

Comment:
is it a breaking change? Before the data used to start with l1Sender, but now it starts with amount